decidim-core/app/controllers/decidim/pages_controller.rb

```diff
@@ -39,5 +40,31 @@ def promoted_participatory_processes
     def highlighted_participatory_processes
       @promoted_processes ||= OrganizationParticipatoryProcesses.new(current_organization) | HighlightedParticipatoryProcesses.new
     end
+
+    private
+
+    def accepted_proposals_count
```

Comment:
Not sure if we should put this on `PagesController`. Maybe we should have a separate controller for the home page instead.